runtime/vampir/stdlib.pir

```diff
@@ -0,0 +1,47 @@
+def isBool x = (x * (x - 1) = 0);
```

Comment:
I think a reference to this file needs to be added to the `extra-source-files` entry in the `package.yaml`:

https://github.com/anoma/juvix/blob/02e2f9f2dd093c504f132b832a0a1ef28b5c1f07/package.yaml#L22

This doesn't seem to be an issue for the stack build, but I think matters for `cabal install` and for other packaging purposes.

Reply:
done